apps/ledger-live-mobile/e2e/specs/receive/currencies.spec.ts

```diff
@@ -13,6 +13,25 @@ describe("Receive different currency", () => {
     await app.init({
       userdata: "skip-onboarding",
       knownDevices: [knownDevice],
+      featureFlags: {
+        llmModularDrawer: {
+          enabled: true,
+          params: {
+            add_account: false,
+            live_app: false,
+            live_apps_allowlist: [],
+            live_apps_blocklist: [],
+            receive_flow: true,
+            send_flow: false,
+            enableModularization: false,
+            searchDebounceTime: 300,
+            backendEnvironment: "STAGING",
+          },
```

Comment:
Actually we forced FF to disabled [here](https://github.com/LedgerHQ/ledger-live/blob/2241fd6f6f30f0125064334cfd3fa75c62f23714/apps/ledger-live-mobile/e2e/page/index.ts#L91) as it was not supported by mocked tests. So removing it, you wouldn't need to force within the tests.